1. debug build (not sure why...)
2. build shared library
3. add optional static lib target
4. Don't install COPYING

diff -urN a/Makefile b/Makefile
--- a/Makefile	2019-05-20 17:16:59.168724034 -0500
+++ b/Makefile	2019-05-20 17:20:12.860726606 -0500
@@ -3,7 +3,7 @@
 -include user.make
 
 ifndef build
-  build := release
+  build := debug
 endif
 
 ifndef OUT
@@ -187,13 +187,15 @@
 
 # --- Library ---
 
-MUPDF_LIB = $(OUT)/libmupdf.a
-THIRD_LIB = $(OUT)/libmupdf-third.a
+MUPDF_LIB = libmupdf.so.${PV}
+MUPDF_STATIC = $(OUT)/libmupdf.a
+THIRD_LIB =
 THREAD_LIB = $(OUT)/libmupdf-threads.a
 PKCS7_LIB = $(OUT)/libmupdf-pkcs7.a
 
-$(MUPDF_LIB) : $(MUPDF_OBJ)
-$(THIRD_LIB) : $(THIRD_OBJ)
+$(MUPDF_LIB): $(MUPDF_OBJ) $(THIRD_OBJ)
+	$(QUIET_LINK) $(CC) $(LDFLAGS) --shared -Wl,-soname -Wl,$(MUPDF_LIB) -Wl,--no-undefined -o $@ $^ $(THIRD_LIBS) $(LIBS)
+$(MUPDF_STATIC): $(MUPDF_OBJ) $(THIRD_OBJ)
 $(THREAD_LIB) : $(THREAD_OBJ)
 $(PKCS7_LIB) : $(PKCS7_OBJ)
 
@@ -322,7 +324,7 @@
 
 	install -d $(DESTDIR)$(docdir)
 	install -d $(DESTDIR)$(docdir)/examples
-	install -m 644 README COPYING CHANGES $(DESTDIR)$(docdir)
+	install -m 644 README CHANGES $(DESTDIR)$(docdir)
 	install -m 644 docs/*.html docs/*.css docs/*.png $(DESTDIR)$(docdir)
 	install -m 644 docs/examples/* $(DESTDIR)$(docdir)/examples
 
